frontend/app/components/contextual-alert.tsx

```diff
@@ -0,0 +1,25 @@
+import type { ReactNode } from 'react';
+
+export interface ContextualAlertProps {
+  message_heading: string;
+  children: ReactNode;
+  type: 'warning' | 'success' | 'danger' | 'info';
+}
+
+export function ContextualAlert(props: ContextualAlertProps) {
+  const { message_heading, children, type } = props;
+
+  const alert_color = type === 'warning' ? 'border-l-amber-700' : type === 'danger' ? 'border-l-red-700' : type === 'info' ? 'border-l-cyan-700' : 'border-l-green-700';
+
+  return (
+    <div id="contextual-alert-id" className={`relative min-w-[290px] pl-[16px] sm:pl-[24px]`}>
```

Comment:
Having a static `id` would mean that if we have two (or more) `<ContextualAlert />` on a single page, there would be duplicate `id`s, which would fail HTML validation. I suggest either allowing the consumer to pass their own `id` property or removing the use the `id` altogether.